src/SFA.DAS.ApplyService.InternalApi/Controllers/ExperienceAndAccreditationController.cs

```diff
@@ -32,24 +32,31 @@ public async Task<string> GetOfficeForStudents(Guid applicationId)
         [HttpGet("/Accreditation/{applicationId}/InitialTeacherTraining")]
         public async Task<InitialTeacherTraining> GetInitialTeacherTraining(Guid applicationId)
         {
-            var initialTeacherTrainingTask = _qnaApiClient.GetAnswerValue(applicationId,
+            var initialTeacherTraining = await _qnaApiClient.GetAnswerValue(applicationId,
                 RoatpWorkflowSequenceIds.YourOrganisation,
                 RoatpWorkflowSectionIds.YourOrganisation.ExperienceAndAccreditations,
                 RoatpWorkflowPageIds.ExperienceAndAccreditations.InitialTeacherTraining,
                 RoatpYourOrganisationQuestionIdConstants.InitialTeacherTraining);
 
-            var isPostGradTrainingOnlyApprenticeshipTask = _qnaApiClient.GetAnswerValue(applicationId,
-                RoatpWorkflowSequenceIds.YourOrganisation,
-                RoatpWorkflowSectionIds.YourOrganisation.ExperienceAndAccreditations,
-                RoatpWorkflowPageIds.ExperienceAndAccreditations.IsPostGradTrainingOnlyApprenticeship,
-                RoatpYourOrganisationQuestionIdConstants.IsPostGradTrainingOnlyApprenticeship);
-
-            await Task.WhenAll(initialTeacherTrainingTask, isPostGradTrainingOnlyApprenticeshipTask);
+            if (initialTeacherTraining.ToUpper() == "YES")
+            {
+                var isPostGradTrainingOnlyApprenticeship = await _qnaApiClient.GetAnswerValue(applicationId,
+                    RoatpWorkflowSequenceIds.YourOrganisation,
+                    RoatpWorkflowSectionIds.YourOrganisation.ExperienceAndAccreditations,
+                    RoatpWorkflowPageIds.ExperienceAndAccreditations.IsPostGradTrainingOnlyApprenticeship,
+                    RoatpYourOrganisationQuestionIdConstants.IsPostGradTrainingOnlyApprenticeship);
+
+                return new InitialTeacherTraining
+                {
+                    DoesOrganisationOfferInitialTeacherTraining = true,
+                    IsPostGradOnlyApprenticeship = isPostGradTrainingOnlyApprenticeship.ToUpper() == "YES"
+                };
+            }
```

Comment:
Would it have been easier to just change isPostGradTrainingOnlyApprenticeshipTask.Result.ToUpper() == "YES" to isPostGradTrainingOnlyApprenticeshipTask.Result?.ToUpper() == "YES"

Reply:
No, because the QnA API responds with a 404 error when the question doesn't have an answer.

Reply:
Gotcha.  Is that 404 consistent?  Lots of the other methods use the null check on the result and they seem to be working?

Reply:
It's worth checking, changes to logging/exception handling mean that anything that's not a successful response is thrown